x-pack/legacy/plugins/integrations_manager/server/integrations/index.ts

```diff
@@ -4,5 +4,44 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-export * from './data';
+import { ClusterClient, ScopedClusterClient } from 'src/core/server/';
+import { AssetTypes, InstallationStatus } from '../../common/constants';
+import { Installable, Installation, Request } from '../../common/types';
+
+export * from './get';
+export * from './install';
+export * from './remove';
 export * from './handlers';
+
+export type CallESAsCurrentUser = ScopedClusterClient['callAsCurrentUser'];
+
+enum SavedObjectTypes {
```

Comment:
These are the types of assets which use Saved Objects (e.g. *not* `ingest-pipeline`)